apps/bare-expo/ios/Podfile

```
@@ -39,6 +40,8 @@ abstract_target 'BareExpoMain' do
       )
     end
 
+    aggregate_xc_privacy!(installer)
```

Nit: why `xc_privacy`?
```suggestion
    aggregate_xcprivacy!(installer)
```